packages/buefy-next/src/components/upload/Upload.vue

```diff
@@ -189,9 +176,13 @@ export default {
                     }
                 }
             }
-            if (!valid) this.$emit('invalid')
+            if (!valid) this.$emit('invalid', valid)
```

Comment:
Why would you add a new parameter to the event? Please do not change the behavior unless there is a problem.

Reply:
Typescript type-check throws an error if I don't add the second parameter
![image](https://github.com/user-attachments/assets/e2dd1e59-b1e1-402c-b9eb-8069bfa987ac)

Reply:
Please see <https://github.com/ntohq/buefy-next/pull/269/files#r1733004271>